### ISSUE TITLE
Labels not included in encodings output

### ISSUE THREAD

**Issue by timmyers15** [fidelity/seq2pat#50]
Hi. This library is very intriguing! I've been able to install and use the library with some of my own data. I've been able to replicate the example outputs from your example notebooks. So far, so good.

When I look at the output created in the encodings (ie the output from a line such as this: _encodings = pat2feat.get_features(sequences, dpm_patterns, drop_pattern_frequency=False)_, I see the sequence with all of the respective one hot columns for the pattern features identified. But here is where I'm stuck. This output doesn't include the original label of each individual example. How do I add the label to this output for later machine learning tasks? Do I need to subsequently join this output back to my original dataset on the sequence column? Or, is there a way to "carry forward" the respective labels throughout the DPM processing through and into this final output?

Thanks for a great library- Tim

**Comment by skadio** [comment]
(heads-up; reading on this mobile) 

my immediate reaction is you are right about subsequent join the generated features and labels. We want to keep patterns, labels, features, distinctive from each other. (think like how sklearn what's you to fit (X, y)) 

IF I remember correct, the row ordering of all of these should be in sync.. That is, the first pattern row corresponds to ther first feature row, and it corresponds to the first label. So we have one-to-one mapping between a pattern, p, its one-hot feature vector, x, and its label. y. In the downstream, we can build ML models on (X,y)

**Comment by timmyers15** [comment]
Thanks for the quick reply! So I downloaded the original data with labels separately from the encoding output and compared the sequences from each. Indeed they appear to be the same row order. I presumed the features records from encoding would have been ordered differently during the DichotomicAggregation looking at unique Pos and Neg subsets separately, but good to know that is not the case. This makes the join back to the original data with labels very straightforward on simple indexes. 

Thanks again for the insight, much appreciated!
